go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/hashicorp/go-secure-stdlib/parseutil v0.1.2
 	github.com/hashicorp/go-secure-stdlib/strutil v0.1.1
 	github.com/hashicorp/vault/api v1.3.0
-	github.com/hashicorp/vault/sdk v0.3.0
+	github.com/hashicorp/vault/sdk v0.3.1-0.20211124134129-09816c37b82e
```

Comment:
Could you link the relevant sdk update that is required for this PR? I noticed that this PR uses `HandlePatchOperation` , which is updated here: https://github.com/hashicorp/vault/pull/13191/files . Was this the change you wanted to include?

Reply:
Ah, yes. I did link the PR above but I will add a note in the description to specify why the sdk version has been updated. Thank you for calling that out!

Reply:
Awesome! In that case, I think we'd want to merge the sdk PR first, then redo the `go get` so we don't depend on a branch.

Reply:
Yup, that's the plan.